### DISCUSSION TITLE
سداد على متجري الالكتروني

### DISCUSSION THREAD

**Discussion by Aisha1025** [woocommerce/woocommerce#48760] [Checkout Flow]
عند تنصيب بوابة الدفع على متجري بلينك لم تقبل تنصيب البوابة ويظهر لي التعليل بان البوابة لاتتوافق مع صفحة سداد

**Comment by barryhughes** [comment]
Via Google Translate:

> When installing the payment gateway on my Blink store, it did not accept the installation of the gateway, and the explanation appeared to me that the gateway was not compatible with the payment page.

**Comment by barryhughes** [comment]
@Aisha1025 have you reached out to the support team for Blink? I'm actually a little unsure what it is you are using ... is it related to [this product/service](https://blinkpayment.atlassian.net/wiki/spaces/IN/pages/113639425/Woocommerce+Integration+Guide)?

**Reply by Aisha1025** [reply]
مع من أتواصل ؟

**Reply by barryhughes** [reply]
I'm not sure what product you are actually using, or who made it, so it's hard for me to answer that.

If it exists as a plugin, you can probably look at the corresponding entry in the plugin admin screen and figure out who to contact from there.
